app/src/compat.h

```diff
@@ -75,6 +75,10 @@
 # define SCRCPY_SDL_HAS_THREAD_PRIORITY_TIME_CRITICAL
 #endif
 
+#if SDL_VERSION_ATLEAST(2, 0, 22)
```

Comment:
Shouldn't it be `(2, 0, 18)`?

https://github.com/libsdl-org/SDL/blob/d0c2d8bc40a90cc1a763f6cf5397e8c9958a33d8/WhatsNew.txt#L210

Reply:
Apparently so.